### DISCUSSION TITLE
revert the change to legal document if the document is empty.

### DISCUSSION THREAD

**Discussion by pederdm000** [tosdr/edit.tosdr.org#1075] [Ideas] [answered]
Also I'm not sure if this is currently implemented but if he document is recrawled that it would revert the change if the document is empty. This would usefull for example if a document has been crawled but is now being blocked from being updated there could be some sort of status indicator showing that it is not currently up to date.

**Comment by JustinBack** [comment, marked as answer]
This is already the case I believe. If the document is empty an error is thrown

**Reply by pederdm000** [reply]
I think you are correct about the error but if i remember correctly some of the serves we had crawled before that use to be in the database go remove as the crawler go blocked and the document got removed and the points  also got removed for the services i think https://edit.tosdr.org/services/325 is one of a few examples i have seen if i remember correctly.
